cmd/build/helmify/static/templates/crds.yaml

```diff
@@ -1,6 +1,319 @@
-{{- if .Values.customResourceDefinitions.create }}
-{{- range $path, $bytes := .Files.Glob "crds/*.yaml" }}
-{{ $.Files.Get $path }}
+{{- if .Values.experimentalEnableMutation }}
```

Comment:
These CRD yamls can be added here: https://github.com/open-policy-agent/gatekeeper/tree/master/manifest_staging/charts/gatekeeper/crds